assets/scripts/create.js

```diff
@@ -0,0 +1,16 @@
+function create(
```

Comment:
Not appropriate function naming
```suggestion
function createSmth(
```